neurom/core/tests/test_soma.py

```diff
@@ -79,15 +79,6 @@
     [11, 22, 33, 44, 2, 1, 1]
 ]
 
-BIFURCATING_SOMA_PTS = [
```

Comment:
h5 files should have bifurcations, since they are contours, I'm mainly fixing the SWC cylinder problem here